Makefile

```diff
@@ -36,6 +36,8 @@ css:
 
 js:
 	mkdir -p $(BUILD)
+        mkdir -p node_modules
```

Comment:
is the mkdir strictly necessary here? I thought npm managed the `node_modules` location and could create it where it needs to based on config / how it is invoked?

Reply:
I'm surprised about this too. An `npm install` here would be more expected.

Reply:
Alright, will move to `olsystem`